dApps.md

```diff
@@ -0,0 +1,47 @@
+<p align="center">
+  <img 
+    src="http://res.cloudinary.com/vidsy/image/upload/v1503160820/CoZ_Icon_DARKBLUE_200x178px_oq0gxm.png" 
+    width="125px"
+  >
+</p>
+
+<h1 align="center">NEO dApps</h1>
+
+<p align="center">
+  A curated list of dApps for Neo, in multiple stages of development.<br/>
+  <i>Last updated: 09/27/17</i>
```

Comment:
I personally don't think the `Last updated` labels are needed, because this is hosted on Github and it's pretty obvious from the history of the file.